src/schema/partner_show_event.ts

```diff
@@ -1,8 +1,46 @@
-import date from "./fields/date"
-import { GraphQLString, GraphQLObjectType } from "graphql"
+import dateField, { date, DateSource } from "./fields/date"
+import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
 import { ResolverContext } from "types/graphql"
 import { exhibitionPeriod } from "lib/date"
 
+const hasOldEmissionUserAgentString = (userAgent: string | string[]): boolean =>
```

Comment:
@anandaroop This typing isn’t correct. You want _just_ `string`, because you’re already filtering out the array below when you invoke this function for each entry in the array. The reason this isn’t failing is because TS thinks you’re possibly looking for an exact match of these strings in the array.

Reply:
😿  with multiple insertion points comes great responsibility

Reply:
It’s still better than implicit `any`!